src/ints/mouse.cpp

```diff
@@ -182,51 +218,65 @@ Bitu PS2_Handler(void) {
 	return CBRET_NONE;
 }
 
-
 #define X_MICKEY 8
 #define Y_MICKEY 8
 
-#define MOUSE_HAS_MOVED 1
-#define MOUSE_LEFT_PRESSED 2
-#define MOUSE_LEFT_RELEASED 4
-#define MOUSE_RIGHT_PRESSED 8
-#define MOUSE_RIGHT_RELEASED 16
-#define MOUSE_MIDDLE_PRESSED 32
-#define MOUSE_MIDDLE_RELEASED 64
 #define MOUSE_DELAY 5.0
 
 void MOUSE_Limit_Events(uint32_t /*val*/)
 {
-	mouse.timer_in_progress = false;
-	if (mouse.events) {
-		mouse.timer_in_progress = true;
+	timer_in_progress = false;
+	if (queue_used) {
+		timer_in_progress = true;
 		PIC_AddEvent(MOUSE_Limit_Events,MOUSE_DELAY);
 		PIC_ActivateIRQ(MOUSE_IRQ);
 	}
 }
 
-inline void Mouse_AddEvent(uint8_t type) {
-	if (mouse.events<QUEUE_SIZE) {
-		if (mouse.events>0) {
+inline void AddEvent(uint8_t type) {
+	if (queue_used < QUEUE_SIZE) {
+		if (queue_used > 0) {
 			/* Skip duplicate events */
-			if (type==MOUSE_HAS_MOVED) return;
+			if (type==DOS_EV::MOUSE_HAS_MOVED || type==DOS_EV::WHEEL_HAS_MOVED) return;
```

Comment:
formatting

Reply:
Fixed. Nevertheless, script will do the rest at the end.